src/assets/custom/css/_sass/_hubspot-ctas.scss

```diff
@@ -0,0 +1,13 @@
+.hubspot-in-page-banner .cta_button {
+  @include cta;
+  @include primary;
+  @include primary-navy;
+}
+
+.hubspot-header-banner-cta .cta_button {
+  @include cta;
+  color: white;
+  padding: 0 !important;
+  border: none;
+  text-decoration: underline !important;
+}
```

Comment:
This looks good. If we create an include for the HubSpot CTAs we could give it a name to match this file. 

I wonder if we can get rid of some of these overrides/resets if we stop including the `hs-cta-wrapper` class name?

Reply:
The hubspot scripts attach a new version of the `hs-cta-wrapper` class with an id fixed onto the end which will override these styles if they don't include the `!important`override.